projects/plugins/jetpack/modules/masterbar/masterbar/class-masterbar.php

```diff
@@ -392,6 +413,79 @@ public function get_locale() {
 		return $wpcom_locale;
 	}
 
+	/**
+	 * Get Jetpack locale name.
+	 *
+	 * @param  string $slug Locale slug.
+	 * @return string Jetpack locale.
+	 */
+	public function get_jetpack_locale( $slug = '' ) {
+		if ( ! class_exists( 'GP_Locales' ) ) {
+			if ( defined( 'JETPACK__GLOTPRESS_LOCALES_PATH' ) && file_exists( JETPACK__GLOTPRESS_LOCALES_PATH ) ) {
+				require JETPACK__GLOTPRESS_LOCALES_PATH;
+			}
+		}
+
+		if ( class_exists( 'GP_Locales' ) ) {
+			$jetpack_locale_object = GP_Locales::by_field( 'slug', $slug );
+			if ( $jetpack_locale_object instanceof GP_Locale ) {
+				$jetpack_locale = $jetpack_locale_object->wp_locale ? $jetpack_locale_object->wp_locale : 'en_US';
+			}
+		}
+
+		return $jetpack_locale;
+	}
+
+	/**
+	 * Install locale if not yet available.
+	 *
+	 * @param string $locale The new locale slug.
+	 */
+	public function install_locale( $locale = '' ) {
+		if ( ! in_array( $locale, get_available_languages(), true )
+			&& ! empty( $locale ) && current_user_can( 'install_languages' ) ) {
+
+			if ( ! function_exists( 'wp_download_language_pack' ) ) {
+				require_once ABSPATH . 'wp-admin/includes/translation-install.php';
+			}
+
+			if ( ! function_exists( 'request_filesystem_credentials' ) ) {
+				require_once ABSPATH . 'wp-admin/includes/file.php';
+			}
+
+			if ( wp_can_install_language_pack() ) {
+				wp_download_language_pack( $locale );
+				load_default_textdomain( $locale );
+			}
+		}
+	}
+
+	/**
+	 * Hide language dropdown on user edit form.
+	 */
+	public function hide_language_dropdown() {
+		add_filter( 'get_available_languages', '__return_null' );
+	}
+
+	/**
+	 * Replace language dropdown with link to WordPress.com.
+	 */
+	public function replace_language_dropdown() {
+		$language_row  = printf( '<table class="form-table" role="presentation"><tbody><tr class="user-language-wrap"><th scope="row">' );
+		$language_row .= printf(
+			'<label for="locale">%1$s<span class="dashicons dashicons-translation" aria-hidden="true"></span></label>',
+			esc_html( __( 'Language', 'jetpack' ) )
```

Comment:
You could use `esc_html__` here:

```suggestion
			esc_html__( 'Language', 'jetpack' )
```

Reply:
Thanks! Changed here: https://github.com/Automattic/jetpack/pull/19525/commits/c2725d02e2f0cb02abddbcd3d401761685ca4edd